Magical Nights/items/enchanted.json

```
@@ -4,20 +4,60 @@
     "type": "GENERIC",
     "symbol": "/",
     "color": "brown",
+    "name": { "str": "minor staff of the magi", "str_pl": "minor staves of the magi" },
+    "description": "A simple carved staff, made of wood.  It faintly glows with magic when you cast spells.  Wielding it slightly increases your mana regeneration",
+    "price": "100 USD",
+    "material": [ "wood" ],
+    "weapon_category": [ "QUARTERSTAVES" ],
+    "flags": [ "SHEATH_SPEAR", "ALWAYS_TWOHAND","FRAGILE_MELEE", "MAGIC_FOCUS" ],
+    "relic_data": {
+      "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "MANA_CAP", "multiply": 0.5 } ] } ]
+    },
+    "weight": "1400 g",
+    "volume": "3 L",
+    "category": "weapons",
+    "to_hit": 3,
+    "bashing": 17
+  },
+  {
+    "id": "magi_staff_lesser",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "brown",
     "name": { "str": "lesser staff of the magi", "str_pl": "lesser staves of the magi" },
     "description": "A beautifully carved staff, made of enchanted wood and mithril.  It faintly glows with magic when you cast spells, but it is not a sturdy melee weapon.",
     "price": "400 USD",
     "material": "wood",
     "flags": [ "SHEATH_SPEAR", "ALWAYS_TWOHAND", "FRAGILE_MELEE", "MAGIC_FOCUS" ],
     "relic_data": {
-      "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "MANA_CAP", "multiply": 0.2, "add": 750 } ] } ]
+      "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "MANA_CAP", "multiply": 1.25} ] } ]
     },
     "weight": "1400 g",
     "volume": "3 L",
     "bashing": 19,
     "category": "weapons",
     "to_hit": 3
   },
+  {
+    "id": "magi_staff_greater",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "brown",
+    "name": { "str": "greater staff of the magi", "str_pl": "greater staves of the magi" },
+    "description": "A beautifully carved staff, made of enchanted wood and mithril.  It faintly glows with magic when you cast spells, but it is not a sturdy melee weapon.  Wielding it greatly increases your mana regeneration.",
```

```suggestion
    "description": "A beautifully carved staff, made of enchanted wood and mithril.  It faintly glows with magic when you cast spells, but it is not a sturdy melee weapon.  Wielding it greatly increases your mana capacity.",
```
As before, pretty sure capacity is what's being increased, not regen.